sources/HUBComponentWrapper.h

```diff
@@ -196,6 +198,12 @@ willUpdateSelectionState:(HUBComponentSelectionState)selectionState;
  */
 - (void)saveComponentUIState;
 
+/** 
+ *  Returns the child component wrapper located at the provided index – if visible. 
+ *  @param index The index of the component to retrieve.
+ */
+- (nullable HUBComponentWrapper *)childComponentAtIndex:(NSUInteger)index;
```

Comment:
Since this is only returning visible components - how about naming it `visibleChildComponentAtIndex:`?